### ISSUE TITLE
Update points prop in type of object list not work

### ISSUE THREAD

**Issue by vvvhung** [djaxho/pure-vue-chart#11]
if prop `points` is type of object list, then update its value will break the chart.
points=[{label: 'N', value: 41.1}, {label: 'NW', value: 1}, {label: 'W', value: 15}]
It seem that component will take the object (eg. {label: 'N', value: 41.1}) as the height value.

I tested some small change and it worked well:
```
    chartData() {
      return this.staticDataPoints.map((dataPoint, index) => ({
        valueNotInMotion: this.tweenedDataPoints[index],
        index,
        partitionWidth: this.partitionWidth,
        midPoint: this.partitionWidth / 2,
        x: index * this.partitionWidth,
        xMidpoint: index * this.partitionWidth + this.partitionWidth / 2,
        yOffset: this.innerChartHeight - this.y(this.tweenedDataPoints[index].value),
        height: this.y(this.tweenedDataPoints[index].value),
      }))
    }
```

**Comment by djaxho** [comment]
@vvvhung cool thanks! Do you want to contribute a pull request to fix it?

Cheers

**Comment by wallcrosstype** [comment]
did pull #23 help this issue ?

**Comment by djaxho** [comment]
fixed by #23
